flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -6,21 +6,19 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt) 
 
-- com.amazonaws:aws-java-sdk-core:1.11.319
-- com.amazonaws:aws-java-sdk-kinesis:1.11.319
-- com.amazonaws:aws-java-sdk-sts:1.11.319
-- com.amazonaws:jmespath-java:1.11.319
-- org.apache.httpcomponents:httpclient:4.5.3
+- com.amazonaws:amazon-kinesis-client:1.11.2
+- com.amazonaws:amazon-kinesis-producer:0.13.1
+- com.amazonaws:dynamodb-streams-kinesis-adapter:1.5.0
```

Comment:
please move this one below the remaining `aws-java` entries so the list is sorted

Reply:
👍 I assume no one will mind if I squash before pushing this change, to avoid another rebase after